docs/sdk/client-side-sdks/react-native/react-native-install.md

```diff
@@ -0,0 +1,117 @@
+The [react-native-device-info](https://www.npmjs.com/package/react-native-device-info) package provides information about the current device running the SDK, which is required to correctly apply targeting rules.
+
+3.  Install the SDK dependencies, run the following command
+
+```bash
+npm install --save @react-native-async-storage react-native-get-random-values react-native-device-info
+```
+or
+
+```bash
+yarn add @react-native-async-storage react-native-get-random-values react-native-device-info
+```
+
+4.  Import the `@react-native-async-storage/async-storage` package somewhere in your code (e.g. in the `App.jsx` file). (see example below)
+5.  Import the `react-native-get-random-values` package somewhere in your code (e.g. in the `App.jsx` file). (see example below)
+6.  Import the `react-native-device-info` package and set `global.DeviceInfo = DeviceInfo`. (see example below)
+
+Example of the above steps:
+```javascript
+import React from 'react'
+import 'react-native-get-random-values'
+import DeviceInfo from 'react-native-device-info'
+import '@react-native-async-storage/async-storage'
+import { withDVCProvider } from '@devcycle/devcycle-react-native-sdk'
+
+global.DeviceInfo = DeviceInfo
+```
+
+7. Wrap your application component tree in either the `withDVCProvider` or `asyncWithDVCProvider` higher-order component (HOC), as explained in the [Getting Started](#getting-started) section.
+
+
+```jsx
+export default withDVCProvider({ sdkKey: '<DVC_CLIENT_SDK_KEY>' })(App)
+```
+
+A complete working example of an `App.jsx` file is below:
+```jsx
+import React from 'react'
+import { View, Text } from 'react-native'
+
+import 'react-native-get-random-values'
+import DeviceInfo from 'react-native-device-info'
+import '@react-native-async-storage/async-storage'
+import { withDVCProvider } from '@devcycle/devcycle-react-native-sdk'
+
+global.DeviceInfo = DeviceInfo
+function App() {
+  return (
+    <View
+      style={{
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <Text>Universal React with Expo</Text>
+    </View>
+  )
+}
+
+export default withDVCProvider({ sdkKey: '<DVC_CLIENT_SDK_KEY>' })(App)
+```
+
+### Provider Config
+
+The `withDVCProvider` function accepts a Provider Config object:
+
+[DVC ProviderConfig Typescript Schema](https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/react/src/types.ts#L3)
+
+| Property | Type | Description            |
+|------------|------|------------------------|
+| sdkKey | string | SDK key                |
+| user | [DVCUser](https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/js/src/types.ts#L55) | DevCycle user object   |
+| options | [DVCOptions](https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/js/src/types.ts#L44) | DevCycle options object |
+
+### Initialization Options
```

Comment:
should we move the Provider Config and Initialization Options in the Getting Started section? I'm also trying to figure out where it goes for the React docs and we should be consistent
because the initialization steps seem to be in the Getting Started section so we should keep them together?

Reply:
yeah was confused about this too, I think we can move it to getting-started as it is the next step after install

Reply:
cool I'll put them into the Getting Started section as well then

Reply:
Worth noting that all the pages are somewhat inconsistent already - I have a followup task to ask Andrew in marketing to do a bit of a review of these docs as well (just need to chat with him). 
But ultimately, if you see an easy small improvement, do it, but don't obsess over it! We can ask marketing to do follow up work on making it pretty/reviewing the organization etc.